keyboards/mechlovin/infinity87/rev2/matrix.c

```diff
@@ -0,0 +1,273 @@
+#include "quantum.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Also, could this be converted to matrix lite? 
https://docs.qmk.fm/#/custom_matrix?id=lite